corehq/ex-submodules/casexml/apps/case/const.py

```diff
@@ -52,3 +51,5 @@
 CASE_INDEX_CHILD = 'child'
 
 DEFAULT_CASE_INDEX_IDENTIFIERS = {CASE_INDEX_CHILD: 'parent', CASE_INDEX_EXTENSION: 'host'}
+
+UNOWNED_EXTENSION_OWNER_ID = '-'
```

Comment:
Oh we should also add this to `WEIRD_USER_IDS` so we don't try to [determine cleanliness on it](https://github.com/dimagi/commcare-hq/blob/master/corehq/ex-submodules/casexml/apps/phone/cleanliness.py#L63)....